pipeline-templates/job/code-build.yml

```
@@ -0,0 +1,45 @@
+jobs:
+- job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
+  variables:
+  - group: BUILD Management Resources
+  steps:
+    - task: gittools.gittools.setup-gitversion-task.gitversion/setup@0
+      displayName: gitversion/setup
+      inputs:
+        versionSpec: 5.x
+
+    - task: gittools.gittools.execute-gitversion-task.gitversion/execute@0
+      displayName: gitversion/execute
+      inputs:
+        useConfigFile: true
+        configFilePath: GitVersion.yml    
+
+    - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-blocks        
+
+    - template: azure-pipelines-templates/build/step/dependency-check.yml@das-platform-building-blocks
+
+    - task: DotNetCoreCLI@2
+      displayName: "Publish Apprentice Portal Web"
+      inputs:
+        command: publish
+        publishWebProjects: false #set to false as this setting (which defaults to true) will parse the entire repo for web projects
+        projects: src/**/*.csproj
```

```suggestion
        projects: src/$(SolutionBaseName)/$(SolutionBaseName).csproj
```
With the SolutionBaseName a variable defined in code-build.yml and SolutionName variable removed from azure-pipelines.yml